Test/PerformanceCollector/FunctionalTests/TestApps/TestApp45/ApplicationInsights.config

```diff
@@ -34,13 +34,10 @@
     </Add>
     <Add Type="Microsoft.ApplicationInsights.Web.ExceptionTrackingTelemetryModule, Microsoft.AI.Web"/>
   </TelemetryModules>
-  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel">
-    <EndpointAddress>http://LocalHost:4554/v2/track</EndpointAddress>
-    <DeveloperMode>true</DeveloperMode>
-  </TelemetryChannel>
+  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel"/>
```

Comment:
This will break the Func tests. (Ai.Config is destroyed after every nuget upgrade) Please revert this file to original.